README.md

```diff
@@ -1,54 +1,235 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Recipe-App] <a name="about-project"></a>
+
+> A Recipe app that keeps track of all your recipes, ingredients, and inventory. You can save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Recipe-App is built with 
 - Ruby on Rails
-- Postgresql
+- PostgresSQL
+- Bootstrap 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
 
-To get a local copy up and running follow these simple steps.
+> Describe between 1-3 key features of the application.
 
-## Getting Started
+- **[Shopping list]**
+- **[Food list]**
+- **[Recipe List]**
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Install and setup PostgreSQL
+- Install and setup Ruby On Rails 
+- Setup Bootstrap
+
+To get a local copy up and running, follow these steps.
+- git clone https://github.com/ogagaoghene/recipe_app
+- cd recipe_app
 
 ### Prerequisites
-- Ruby
-- Postgresql
-- Rails
 
-## Setup
+In order to run this project you need:
 
-### Clone this repository
-git clone https://github.com/ogagaoghene/recipe_app
+- Ruby on Rails 
+- PostgreSQL
+- Bootstrap 
+- npm install 
+- rails server
 
+### Setup
 
-### Run project
+Clone this repository to your desired folder:
+- git clone https://github.com/ogagaoghene/recipe_app
 - cd recipe_app
-- run rails s
+- rails server
+
+### Install
+
+Install this project with:
+
+- Ruby on Rails
+- PostgreSQL
+- Bootstrap
+- npm install
+- gem install rails
+- gem install bundler
+
+### Usage
+
+To run the project, execute the following command:
+
+- rails server or rails s
+
+### Run tests
+
+To run tests, run the following command:
+- bundle exec rspec spec/feature/test_spec.rb
```

Comment:
* Kindly fix these errors i get when i try to run the tests
![image](https://user-images.githubusercontent.com/52497246/203882522-35bd3d0a-f07c-41f4-bb9c-7cba053454d8.png)

Reply:
@Davidosky007 Am I supposed write tests. I did not write any. Those are default test cases. Writing tests is not part of the requirement. Kindly clarify on this please

Reply:
@Davidosky007 I did intend to put that line. But, in the new README file which I saw a line. I just followed through by giving an example of how to write tests. If you check the requirements, writing of tests is not part of it. Could you please revert on this?

Reply:
@Davidosky007 Thanks for your professional code review. I am the only one doing the work of 2 persons. I am exhausted but what can I do. I have fixed the errors with the tests you requested, You can try again by running the command below:
-
bundle exec rspec spec/tests/